app/config/FrontendAppConfig.scala

```diff
@@ -28,9 +28,12 @@ class FrontendAppConfig @Inject() (configuration: Configuration, contactFrontend
 
   val host: String    = configuration.get[String]("host")
   val appName: String = configuration.get[String]("appName")
+  val feedbackLink    = s"${contactFrontendConfig.baseUrl.get}/contact/beta-feedback?service=${contactFrontendConfig.serviceId.get}"
 
-  def feedbackUrl(implicit request: RequestHeader): String =
-    s"${contactFrontendConfig.baseUrl.get}/contact/beta-feedback?service=${contactFrontendConfig.serviceId.get}&backUrl=${SafeRedirectUrl(host + request.uri).encodedUrl}"
+  def feedbackUrl(implicit request: RequestHeader): String = request.uri match {
+    case uri if uri.contains("problem") => s"$feedbackLink"
```

Comment:
Did we get it agreed or discussed with Jill or Ram that it is all pages which with URL containing `problem` should not have back url on the feedback page? We may have a few like `/register/problem/business-not-identified`, `/register/problem/individual-sign-in-problem`, `/register/problem/organisation-without-utr-pre-registered` and a few more.

Reply:
Still waiting on Ram for this one